src/mmw/js/src/analyze/models.js

```diff
@@ -141,89 +141,210 @@ var AnalyzeTaskCollection = Backbone.Collection.extend({
     model: AnalyzeTaskModel
 });
 
-function createAnalyzeTaskCollection(aoi, wkaoi) {
-    var tasks = [
+var AnalyzeTaskGroupModel = Backbone.Model.extend({
+    defaults: {
+        name: 'analysis',
+        displayName: 'Analysis',
+        enabledForCatalogMode: false,
+        tasks: null, // AnalyzeTaskCollection
+        activeTask: null
+    },
+
+    initialize: function() {
+        var tasks = this.get('tasks');
+
+        if (tasks === null || typeof tasks === 'string') {
```

Comment:
What is the use case of `tasks` being a string?

Reply:
There is none - this is taken from some code I cribbed that had a sub-collection model. I'lll remove it.